package.json

```diff
@@ -61,21 +61,23 @@
     "regenerator-runtime": "^0.11.1",
     "sass-loader": "^7.1.0",
     "swagger-tools": "0.9.7",
-    "webpack-dev-server": "2.11.2"
+    "webpack": "^4.27.1",
```

Comment:
I tried everything with this, but there are too many things depending on each other: babel, webpack, webpacker, webpack-dev-server, rails.. something would break. I think after addressing babel/webpack cleanup (#435 ), webpack should be easy.